vllm/model_executor/models/falcon.py

```diff
@@ -406,6 +409,7 @@ def __init__(self, *, vllm_config: VllmConfig, prefix: str = ""):
         config = vllm_config.model_config.hf_config
         quant_config = vllm_config.quant_config
         self.config = config
+        self.config.num_ln_in_parallel_attn = None
```

Comment:
Please remove this line.

Reply:
Sorry, I forget this. Please take a look at the latest updates. Thank you!